.circleci/config.yml

```diff
@@ -473,7 +473,13 @@ jobs:
       - prepare
       - run:
           name: Prime calypso.live build
-          command: ./test/e2e/scripts/wait-for-running-branch.sh
+          command: |
+            if [[ -z $CIRCLE_PR_USERNAME ]]; then
+              ./test/e2e/scripts/wait-for-running-branch.sh
+            else
+              echo 'PRs from external authors cannot run on calypso.live'
+              exit 1
```

Comment:
We could move this condition and `exit 1` as the very first step in this job. That would save the prepare work, most of which is mounting the workspace.